### DISCUSSION TITLE
Welcome to maya-usd Discussions!

### DISCUSSION THREAD

**Discussion by kxl-adsk** [Autodesk/maya-usd#870] [General]
## 👋 Welcome!

   We’re using Discussions as a place to connect with other members of our community. It should give us a much better flow from idea/question -> discussion -> issue or pull request. We hope that you:
  * Ask questions you’re wondering about.
  * Share and discuss ideas.
  * Post announcements about upcoming changes.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * …or simply say `thank you` when someone makes your life easier by providing a fix or feature

**Comment by suesauer** [comment]
Thanks for the hard work so far guys!! Well done.
